reference/checkouts.v3.yml

```diff
@@ -1048,35 +1022,12 @@ definitions:
                               type: number
                               format: double
                             discounted_amount:
-                              description: >-
-                                The discounted amount applied within a given
-                                context.
+                              description: The discounted amount applied within a given context.
                               type: number
                               format: double
                       coupons:
                         description: ''
-                        type: array
```

Comment:
I believe this should be `type: object`. The AppliedCoupon model has all of the applicable properties: id, code, discounted_amount, coupon_type.